app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/viewmodel/EventDetailState.kt

```diff
@@ -8,5 +8,8 @@ data class EventDetailState(
     val event: AgendaItem.Event = AgendaItem.Event(),
     val isEditMode: Boolean = true,
     val navigateBack: Boolean = false,
+    val showTimePicker: Boolean = false,
+    val showDatePicker: Boolean = false,
+    val isFrom: Boolean? = null,
```

Comment:
If you have 2 events (as mentioned on previous comment) you won't need a state variable

Reply:
If I have 2 events ShowFromDatePicker and ShowToDatePicker, then on the selected time/date how can I identify on the viewmodel that on DateSelected/TimeSelected I have to change the "to" or "from" date? (if I remove the isFrom variable)

Reply:
Sorry, in this case you´d have 4 events. Not 2. My bad